frontend/src/features/Categories/categoriesSlice.ts

```diff
@@ -1,10 +1,14 @@
 import { createAsyncThunk, createSlice } from "@reduxjs/toolkit";
 import { RootState } from "../../app/store";
 import { fetchQuiz } from "./categoriesAPI";
-import { Quiz } from "../../Types/Quiz";
 
 export interface QuizState {
-  quizzes: Quiz[];
+  quizzes: {
+    id: number;
+    title: string;
+    description: string;
+    already_taken: string;
+  }[];
```

Comment:
I'm not sure if this is possible pro if you have Quiz type then in your Quiz type you can set already_taken as optional parameter with ? so example
```
type Quiz = {
  id: number,
  title:  string,
  description: string,
  already_taken?: string
}